common/djangoapps/student/helpers.py

```diff
@@ -568,8 +568,8 @@ def _cert_info(user, course_overview, cert_status):
             # who need to be regraded (we weren't tracking 'notpassing' at first).
             # We can add a log.warning here once we think it shouldn't happen.
             return default_info
-
-        status_dict['grade'] = text_type(max(cert_grade_percent, persisted_grade_percent))
+        max_grade = max(filter(lambda x: x is not None, [cert_grade_percent, persisted_grade_percent]))
```

Comment:
This will still fail and throw an exception if both items are None.  I think in that case it should return `None` for consistency.